google_permissions/outputs.tf

```
@@ -5,6 +5,7 @@ locals {
   folder_additional_roles = [
     "roles/bigquery.jobUser",
     "roles/bigquery.dataViewer",
+    "roles/bigquery.resourceAdmin",
```

```suggestion
    "roles/bigquery.resourceAdmin",
```

Is this the correct role we are looking for here? This role is associated with managing slot capacity, which doesn't happen in tenant projects.

If it's for managing and manipulating datasets and tables you might want 
https://cloud.google.com/bigquery/docs/access-control#bigquery.dataEditor

---

I missed the truncated comment, you want to be able to view bigquery job details, roles/bigquery.resourceViewer might be better here.

---

I was going off of https://cloud.google.com/bigquery/docs/managing-jobs#required_permissions_2 which mentions roles/bigquery.resourceAdmin which has the bigquery.jobs.listAll capability. However, roles/bigquery.resourceViewer also has bigquery.jobs.listAll. So if that works then that sounds even better/more granular

---

Updated it to use resourceViewer.